### DISCUSSION TITLE
Is there a way to show a picture 2D with no interaction besides user added functions but still have access to buttons?

### DISCUSSION THREAD

**Discussion by DYT-Flyer** [marcomusy/vedo#540] [Q&A] [answered]
Plotter currently has 'image' mode, but depending on how the user interacts with the image it changes the contrast. I want to be able to add spline points without worrying about affecting the contrast.

**Comment by marcomusy** [comment, marked as answer]
try `mode="2d"` or check if there is any other suitable mode in the plotter docs.
